### ISSUE TITLE
The problem of app_key used in Web Applications

### ISSUE THREAD

**Issue by DyncKathline** [Countly/countly-server#1021]
### More Description
I used app_key for the web application created by countly server, but it wasn't the app_key that was displayed when I sent it.
![image](https://user-images.githubusercontent.com/16282974/62910930-090ba880-bdb5-11e9-88d1-a410d64e043f.png)
![image](https://user-images.githubusercontent.com/16282974/62910958-204a9600-bdb5-11e9-851e-dbae70ec28f0.png)
![image](https://user-images.githubusercontent.com/16282974/62910982-3b1d0a80-bdb5-11e9-93c4-5d4b0a583d75.png)

### Your Environment
window 10
countly server 19.08

**Comment by ar2rsawseen** [comment]
If you used some some other app_key for the same domain before, there could be requests that stuck in persistent storage queue before, and they need to be processed first.

All you can clear the queue via `Countly._internals.clearQueue()`

**Comment by DyncKathline** [comment]
thank you

**Comment by windware-ono** [comment]
I just fixed it by adding,
`Countly.namespace = Countly.app_key`
but this needs fixing as it is very confusing how nothing seems to work unless you figure out that the wrong app_key is being sent from the browser.

**Comment by ar2rsawseen** [comment]
Yes app key based storages are due in this release for SDKs (at least for web)
